index.js

```diff
@@ -0,0 +1,65 @@
+const foregroundColor = document.getElementById("foreground-color");
+const backgroundColor = document.getElementById("background-color");
+const ratioResult = document.getElementById("contrast-ratio-result");
+
+const hexToRGB = (hexColor) => {
+    const R = parseInt(hexColor.slice(1, 3), 16);
+    const G = parseInt(hexColor.slice(3, 5), 16);
+    const B = parseInt(hexColor.slice(5), 16);
+    return [R, G, B];
+  }
+  
+const getLuminance = (RGBarray) => {
+    //convert 8bit colors to
+    //RsRGB, GsRGB BsRGB
+    const XsRGBarray = RGBarray.map(elem => elem / 255);    
+    
+    const getRGBCoefficient = (XsRGB) => {
+      if (XsRGB <= 0.03928) {
+          return XsRGB / 12.92;
+      }
+      else {
+          return Math.pow((XsRGB + 0.055) / 1.055, 2.4);
+      }
+    }
+
+    //array of multipliers, for r, g, b respectively
+    //L = 0.2126 * R + 0.7152 * G + 0.0722 * B
+    const multipliers = [0.2126, 0.7152, 0.0722];
+
+    let luminance = 0;
+
+    for (let i = 0; i < multipliers.length; i++) {
```

Comment:
You can consider optimizing your loop so multipliers.length isn't calculated for each loop.

Reply:
Good catch!
However, this PR is already merged.
So it's not my loop, it's kinda our loop already.
You can create new issue for that.